src/components/Project/ProjectNew.js

```diff
@@ -0,0 +1,300 @@
+    super(props);
+    this.state = {
+      clientSelect: '',
+      client_id: '',
+      title: '',
+      summary: '',
+      start_date: '',
+      end_date: '',
+      participantSelect: [],
+      participants: [],
+      clients: [],
+      users: [],
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleClientChange = this.handleClientChange.bind(this);
+    this.handleParticipantChange = this.handleParticipantChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('https://lesewert.herokuapp.com/api/v1/users')
+      .then(res => res.json())
+      .then(data =>
+        this.setState({
+          users: data.users,
+        }),
+      )
+      .catch(() => console.log('error'))
+      .then(
+        fetch('https://lesewert.herokuapp.com/api/v1/clients')
+          .then(res => res.json())
+          .then(data =>
+            this.setState({
+              clients: data.clients,
+            }),
+          )
+          .catch(() => console.log('error')),
+      );
+  }
+
+  handleChange(e) {
+    e.preventDefault();
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  }
+
+  handleClientChange(option) {
+    this.setState({
+      clientSelect: option,
+    });
+  }
+
+  handleParticipantChange(option) {
+    this.setState({
+      participantSelect: option,
+    });
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.clientSelect !== this.state.clientSelect) {
+      this.setState({
+        client_id: this.state.clientSelect.value,
+      });
+    } else if (prevState.participantSelect !== this.state.participantSelect) {
+      this.setState({
+        participants: [
+          ...this.state.participants,
+          this.state.participantSelect[this.state.participantSelect.length - 1]
+            .value,
+        ],
+      });
+    }
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    fetch('/api/v1/projects', {
+      method: 'POST',
+      headers: new Headers({
+        'Content-Type': 'application/json',
+      }),
+      body: JSON.stringify(this.state),
+    })
+      .then(res => {
+        if (res.status >= 200 && res.status < 300) {
+          console.log(res);
+          this.props.history.push('/projects');
+          return res;
+        } else {
+          console.log('Somthing happened wrong');
+        }
+      })
+      .catch(err => err);
+    this.setState({
+      client_id: '',
+      title: '',
+      summary: '',
+      start_date: '',
+      end_date: '',
+      participants: '',
+    });
+  }
+
+  render() {
+    let clientOptions = this.state.clients.map(client => {
+      return {value: client.id, label: client.name};
+    });
+    let participantOptions = this.state.users.map(user => {
+      return {value: user.id, label: user.name};
+    });
+    console.log(this.state);
+    return (
+      <div className='main-panel'>
+        <div className='content'>
```

Comment:
I suggest to move these lines into the page ProjectNew file, so that if more components inside this page get added, it wont be a problem. and for consistency, example:
`    <React.Fragment>
      <Header />
      <div className='main-panel'>
        <div className='content'>
          <ProjectAdd />
          <ProjectTable />
        </div>
      </div>
    </React.Fragment>`

Reply:
Sorry it does not format my reply correctly, you can look at the Project.js file